src/compute-pr-actions.ts

```diff
@@ -302,7 +302,7 @@ export function process(prInfo: BotNotFail,
 
     // Ping reviewers when needed
     if (!(info.hasChangereqs || info.approvedBy.includes("owner") || info.approvedBy.includes("maintainer"))) {
-        if (info.noOtherOwners) {
+        if (info.otherOwners.length === 0) { // Including PRs without any owners

```

Comment:
Re the above comment: this *is* already such a place where you should be aware of the subtle suggested meaning of `noOtherOwners`.

Reply:
`noOtherOwners` does sound like there is *an* owner, but subtle for sure.